lib/rbeapi/api/staticroutes.rb

```diff
@@ -38,17 +38,18 @@ module Rbeapi
   # Rbeapi::Api
   module Api
     ##
-    # The Staticroute class provides a configuration instance for working
+    # The Staticroutes class provides a configuration instance for working
     # with static routes in EOS.
     #
-    class Staticroute < Entity
+    class Staticroutes < Entity
       ##
       # Returns the static routes configured on the node
       #
       # @example
       #   {
-      #     <route>: {
-      #       "next_hop": <string>,
+      #     <route_dest/masklen/next_hop>: {
```

Comment:
No need to generate a unique identifier. That is a puppet requirement not an rbeapi. This library is taking the output from the show command and formatting. If a show command were to return duplicate entries, not that it would, that would be the behavior of EOS and the duplicate entries should be passed on to the caller.